content/geoip/release-notes/2024.mdx

```diff
@@ -77,7 +77,7 @@ and branding conventions:
 | `Versatel Deutschland`              | `1&1 Versatel`                   |
 | `Telefonica O2 UK`                  | `O2 UK`                          |
 | `INDOSAT Internet Network Provider` | `Indosat`                        |
-| `Telefonica del Peru`               | `Telefonica Peru`                |
+| `Telefonica del Peru`               | `Movistar Peru`                  |
```

Comment:
Nitpick, but there's some additional white space after `Movistar Peru`

Reply:
so i added it to make the vertical lines at the end line up, they didn't until I did that -- I checked and they all have extra spacing added :)

Reply:
i've just had another change come through for this same release note so i'm going to add it to this PR if that's okay, we can review again tomorrow.

Reply:
@dhogan8 this is ready for another look when you have a bit of time :) Thanks!

Reply:
Oops! My bad and good catch.

Reply:
not at all, i appreciate it!! thanks!!